internal/migrations/migrate_collection.go

```diff
@@ -13,6 +14,17 @@ func MigrateCollectionSpec(spec logging.ClusterLoggingSpec) logging.ClusterLoggi
 		return spec
 	}
 
+	if spec.Visualization == nil && spec.LogStore != nil && spec.LogStore.Type == logging.LogStoreTypeLokiStack {
+		log.V(3).Info(fmt.Sprintf("Migrating: visualisation spec not set but LogStore is %s, going to set %s as default visualisation spec", spec.LogStore.Type, logging.VisualizationTypeOCPConsole))
+		spec.Visualization = &logging.VisualizationSpec{
+			Type: logging.VisualizationTypeOCPConsole,
+			OCPConsole: &logging.OCPConsoleSpec{
```

Comment:
If this is not a required field, I would refrain from defining them and allow the defaults to take precedence

Reply:
ok